### DISCUSSION TITLE
l got Kratos Binaries for Windows ,but meet some usage question

### DISCUSSION THREAD

**Discussion by W-Sinbad** [KratosMultiphysics/Kratos#11864] [Q&A]
![1701296740584](https://github.com/KratosMultiphysics/Kratos/assets/120139821/e18ed3b1-6175-4e35-981c-eeff6e20e13b)
![1701296787901](https://github.com/KratosMultiphysics/Kratos/assets/120139821/b480004e-09e2-42d1-822e-dd6abccf2b52)

**Comment by rubenzorrilla** [comment]
The version you are running with is rather old (7.0.0). In those days the `LinearSolversApplication` didn't exist (that's why the module error is thrown). I'd try to download latest release.
